### DISCUSSION TITLE
底座底部存在滚动条，如下显示，样子呲了

### DISCUSSION THREAD

**Discussion by congcongnalian** [he3-app/he3-feedback#198] [Bug Report]
### What is expected?
![image](https://github.com/he3-app/he3-feedback/assets/68501045/80ec5ebd-80c0-4c34-8617-7990feefa9b2)

**Comment by lyzhang1999** [comment]
感谢您的反馈，我已经建立了 issues，您可以随时关注。https://github.com/he3-app/he3-feedback/issues/207
